graphics/rendering/arnold/arnold.def

```diff
@@ -0,0 +1,25 @@
+Bootstrap: library
+From: debian:9
+
+%environment
+    # We will install arnold to /opt/arnold
+    PATH=/opt/arnold/bin:$PATH
+    export PATH
+
+    # If you have a license server, you can set this
+    #  -- https://support.solidangle.com/display/A5AILIC/Setting+solidangle_LICENSE+on+the+command+line
+    # solidangle_LICENSE=5053@server
+    # export solidangle_LICENSE
+
+%post
+    apt-get update
+
+    # Required by Arnold
+    apt-get -y install python
+    apt-get -y install less
+    apt-get -y install libx11-6
```

Comment:
Why all on different lines?

Reply:
Because, initially I found I needed python for the installer ... then the installer called less ... and then if you run kick without `-dw` it puts up a render window, so you need the X11 libs ...

Makes perfect sense...